frontend/src/app/shared/types/posts/post-get.model.ts

```diff
@@ -7,6 +7,7 @@ export interface PostGet {
   content: string;
   author: UserGet;
   createdAt: Date;
+  modifiedAt: Date | null;
```

Comment:
w sumie chyba lepiej byłoby dać `modifiedAt?: Date;`

Reply:
wiesz co jak tak robie to nie bardzo działa

Reply:
Jest potem problem z porównywaniem wartości do nulla, wiec chyba musi tak zostac

Reply:
aaa bo wtedy zamiast `value === null` dajesz `!value`

Reply:
i tak jest chyba lepiej bo ci złapie jak będzie undefined

Reply:
zrobiłem tak tylko teraz jest problem, ze zmienna modifiedAt nie moze byc null

Reply:
czyli Api tez powinno zwrocic undefined

Reply:
Dobra zostaw tak jak było i guess

Reply:
dobra